packages/tslint.json

```diff
@@ -1,33 +1,51 @@
 {
+  "defaultSeverity": "warning",
   "rules": {
     "class-name": true,
-    "comment-format": [true, "check-space"],
+    "comment-format": [
+      true,
+      "check-space"
+    ],
     "curly": true,
     "forin": true,
-    "indent": [true, "spaces"],
-    "max-line-length": [true, 180],
+    "indent": [
+      true,
+      "spaces"
+    ],
+    "max-line-length": [
+      true,
+      180
+    ],
     "no-trailing-whitespace": true,
-    "no-unused-expression": true,
-    "no-use-before-declare": true,
```

Comment:
I guess we remove this rule because it's mostly applicable to "var", which we disallow in the following rule?